### PR TITLE
fix: Create gitignore for node_modules and other files

### LINKED ISSUE #1: No .gitignore file
This project does not have a `.gitignore`. This will cause trouble developers contributing to the repo who wish to add files using `git add -A`. I assume not all developers have a global `.gitignore`.

I suggest using a `.gitignore` file like the Node client:
https://github.com/googleapis/google-api-nodejs-client/blob/master/.gitignore

### PR DESCRIPTION
**Pull request by grant**
Creates a `.gitignore` file.
Fixes #1.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+node_modules/
+build/
+/docs/
+.coverage/
+npm-debug.log
+.nyc_output
+.vscode
+.idea
+.DS_Store
+package-lock.json
```
Comment (swalkowski) [inline .gitignore]: We want package-lock.json to be checked in. I don't think it should be ignored then?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+yarn.lock
```
Comment (swalkowski) [inline .gitignore]: Let's ignore yarn-error.log as well.

Comment (grant) [reply .gitignore]: Done.

File: .gitignore (hunk 1 above)
Comment (grant) [reply .gitignore]: Correct. Reverted.

**Pull request merged**